dotCMS/src/main/resources/dotmarketing-config.properties

```diff
@@ -835,4 +835,7 @@ analytics.accesstoken.renewjob.cron=0 0/1 * * * ?
 
 
 ## Favorite Pages Portlet
-DOTFAVORITEPAGE_FEATURE_ENABLE=false
\ No newline at end of file
+DOTFAVORITEPAGE_FEATURE_ENABLE=false
+
+## New Template Builder Portlet
+FEATURE_FLAG_TEMPLATE_BUILDER_2=false
```

Comment:
Should this start with `DOT_` @fmontes ?

Reply:
@wezell said in #lunik `FEATURE_FLAG_{feature_name}`